docker-compose.yml

```diff
@@ -63,13 +62,37 @@ services:
 
   qpc-celery-worker:
     build: .
-    entrypoint: bash ./deploy/entrypoint_celery_worker.sh
+    entrypoint:
+      - /bin/bash
+      - -c
+      - >-
+        pip install debugpy;
```

Comment:
Have you confirmed that `debugpy` works as expected in the Celery worker container here?

Reply:
> Have you confirmed that debugpy works as expected in the Celery worker container here?
not yet

Reply:
good idea about failing fast! will change it.

Reply:
![image](https://github.com/quipucords/quipucords/assets/6073215/7a7a4f29-efe8-42eb-836a-9f89f6105d1b)